files/context_processors.py

```diff
@@ -31,4 +31,6 @@ def stuff(request):
     ret["ALLOW_RATINGS_CONFIRMED_EMAIL_ONLY"] = settings.ALLOW_RATINGS_CONFIRMED_EMAIL_ONLY
     ret["VIDEO_PLAYER_FEATURED_VIDEO_ON_INDEX_PAGE"] = settings.VIDEO_PLAYER_FEATURED_VIDEO_ON_INDEX_PAGE
     ret["RSS_URL"] = "/rss"
+    ret["LIVESTREAM_BACKEND"] = settings.LIVESTREAM_BACKEND
```

Comment:
you're not adding a default value on settings, so this will make the process break with 
```
AttributeError: 'Settings' object has no attribute 'LIVESTREAM_BACKEND'
```

Better wrap this in a try/except to ensure that it does not break

Reply:
Added in latest commit